src/readii/negative_controls_refactor/regions.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+from .abstract_classes import RegionStrategy
+
+
+class FullRegion(RegionStrategy):
+	"""Region strategy to apply control to the entire image."""
+
+	region_name = "full"
+
+	def __call__(self, image_array: np.ndarray, mask_array: np.ndarray) -> np.ndarray:
```

Comment:
`image_array` isn't actually used in any of the region strategies. Can it be removed?